src/Pages/2DPrinting/2DPrintingHealthCheck.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import { healthCheck } from '../../APIFunctions/2DPrinting';
+import {
+  Spinner,
+  Container
+} from 'reactstrap';
+
+export default function PrintingHealthCheck(props) {
+  const [loading, setLoading] = useState(true);
+  const [printerHealthy, setPrinterHealthy] = useState(false);
+
+  useEffect(() => {
+    async function checkPrinterHealth() {
```

Comment:
This function should be declared and implemented outside of useEffect. Removes the clutter in useEffect function